backend/src/main/java/com/redhat/cloud/notifications/processors/email/RecipientResolver.java

```diff
@@ -36,11 +35,11 @@
         }
 
         return Multi.createFrom().iterable(endpoints)
-                .onItem().transformToUni(e -> recipientUsers(accountId, e, subscribers))
-                .concatenate().collect().in(HashSet<User>::new, Set::addAll);
+                .onItem().transformToUni(e -> recipientUsers(accountId, subscribers))
+                .concatenate().collect().in(HashSet::new, Set::addAll);
     }
 
-    private Uni<Set<User>> recipientUsers(String accountId, Endpoint endpoint, Set<String> subscribers) {
+    private Uni<Set<User>> recipientUsers(String accountId, Set<String> subscribers) {
```

Comment:
Please put the `endpoint` arg back here to avoid creating a conflict in one of @josejulio's PRs. This arg which is not currently used will be used very soon.

Reply:
it' back again

Reply:
Thanks :smiley: